app/views/posts/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="columns is-centered">
+  <section class="hero is-white">
+    <div class="hero-body">
+      <div class="container">
+        <% if user_signed_in? %>
```

Comment:
Move logic into view helpers

Reply:
This is the same code from your first comment. So, my question remains same. Plus, what exactly is a "view helper"? I tried Googling, not a clear idea about it yet...